dotcom-rendering/src/web/lib/useApi.tsx

```diff
@@ -27,12 +27,14 @@ interface ApiResponse<T> {
  * returns { loading, error, data }
  * @param {String} url - The url to fetch
  * @param {SWRConfiguration} options - The SWR config object - https://swr.vercel.app/docs/options
+ * @param {RequestInit} init - The fetch init object - https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options
  * */
 export const useApi = <T,>(
```

Comment:
That dangly comma, though 👀

Reply:
🤷

Reply:
I think it’s TS not playing nice with TSX. I’d recommend using using `T extends unknown` instead, which is more informative of what the type of `T` can be. More suggestions below, such as using `Data` instead of `T`.

Reply:
👍  See: https://github.com/guardian/dotcom-rendering/pull/3840